### PR TITLE
fix  crash on rrule.between returned bad dates #3256

### PR DESCRIPTION
**Pull request by sdetweil**
Fixes: #3256 

BUT.. the testcase is inconclusive.. as the code FAILS without the fix, BUT somehow RETURNS 0 entries..
in real life run the node helper fails, and all calendar processing stops.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -30,7 +30,8 @@ _This release is scheduled to be released on 2024-01-01._
 - Fix yr API version in locationforecast and sunrise call (#3227)
 - Fix cloneObject() function to respect RegExp (#3237)
 - Fix newsfeed module for feeds using "a10:updated" tag (#3238)
-- Fix issue template (#3167)
+- Fix issue template (3167)
```
Comment (rejas) [inline CHANGELOG.md:33]: please re-add the # :-)

File: tests/configs/modules/calendar/bad_rrule.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+/* MagicMirror² Test config default calendar with auth by default
+ *
+ * By Rodrigo Ramírez Norambuena https://rodrigoramirez.com
```
Comment (rejas) [inline tests/configs/modules/calendar/bad_rrule.js:3]: you can put your own name here

**Review by rejas — CHANGES_REQUESTED** [review]

File: .prettierignore [hunk 3]
```diff
@@ -1,3 +1,6 @@
+.eslintignore
+.prettierignore
```
Comment (KristjanESPERANTO) [inline .prettierignore:2]: Why should prettier ignore these two files?

**Comment by sdetweil** [comment]
because they produced errors in prettier about no ruleset.  same as the ics

File: .prettierignore (hunk 3 above)
Comment (khassel) [reply .prettierignore:2]: these changes are meanwhile already on `develop`

Comment (sdetweil) [reply .prettierignore:2]: yes, I took them from the other calendar fix

**Review by khassel — APPROVED** [review]

**Review by rejas — APPROVED** [review]

**Pull request merged**